test/e2e/xhr.spec.ts

```diff
@@ -113,10 +113,10 @@ describe('xhr', () => {
     })
     it('should support with url', async () => {
       engine = new LiquidUMD({
-        root: 'https://foo.com/bar/',
+        root: 'http://foo.com/bar/',
```

Comment:
This, like mywebsite.com, is another example where `example.com` or a friend should really be used. `foo.com` isn't owned by liquidjs and could be malicious at any time. There were 8 instances of `foo.com` and only 2 were using `https:`.